### ISSUE TITLE
Meta-issue: track Licensing for 3.0

### ISSUE THREAD

**Issue by swinslow** [spdx/spdx-spec#386]
This is a meta-issue to track all the licensing-related issues for the 3.0 spec release. Not all of these issues are agreed to be added to the spec; many require further discussion and consideration.

Comments on the issues below should be handled in the applicable threads, NOT in this meta-issue. Feel free to comment here re: issues that are missing from this tracking list.

#### Main issue
- [x] (Consolidate fields for license profile, #385

#### Fields for licensing profile
- [X] FOSSology requests:
  - [X] SPDX coverage from what FOSSology covers, licensing-related sub-issues either fixed or wontfix, see #112 
  - [X] ~~Obligations, Risks, Restrictions part of the software delivery in B2B case~~, closed / wontfix, see #111 
  - [X] ~~Adding attribute of distribution relevance at license findings / files / folders~~, closed / wontfix, see #92 
- [X] Introduce CopyrightText at all levels and relax cardinality, #36

#### License expression syntax changes
- [X] ~~Expand License namespaces available~~, closed / wontfix, see #113 
- [X] Support creation of custom exception using "ExceptionRef", #153, added in 3.0 with "AdditionRef" as prefix
- [X] ~~Consider whether '+' should work for Exception IDs too~~, closed / wontfix, see #346 
- [X] ~~License expression semicolon operator~~, closed / wontfix, see #123 
  - [X] ~~_Related:_ Multiple licenses: unclear specification~~, closed / wontfix, see #124 (also #123)
- [X] Simplify license expression grammar in Appendix IV, #62 
- [ ] Add "NOASSERTION" to the license expression syntax, #50 => **moved to 3.1**
- [ ] Add "NONE" to the license expression syntax, #49 => **moved to 3.1**

#### Spec syntax / appendix changes
- [X] Incompatibility in comment property for license between JSON and RDF/XML, #158 
- [X] ~~Rename SPDX-License-Identifier to SPDX-License-Expression?~~, no changes, see #82
- [X] Add appendix describing SPDX Listed License fields, #46
- [ ] Make the matching template formats of license part of the spec - add matching guidelines annotation to SPDX licenses and to NON-SPDX licenses, #32 => **moved to 3.1**

#### License List changes
- [X] Add false "isFsfLibre" value, #93
- [ ] Need to be able to describe relationships between SPDX license-list files (new element?), #13 => **moved to 3.1 but may close or move to license-list-XML**

#### Relationship / Annotation changes
- [X] ~~Expand §8.3.4 Annotation Type with new values (LICENSE, PATENT, COPYRIGHT, EXPORT, TRADEMARK) and change cardinality~~, no changes for licensing-related fields, see #35

#### License for SPDX data
- [x] ~~Re-evaluating CC0-1.0 as DataLicense for SPDX 3.0~~, no changes, see #159

**Comment by jlovejoy** [comment]
#82 is now closed

**Comment by mxmehl** [comment]
I just found this meta issue, and was wondering whether #502 and #464 (a PR but actually an open issue/discussion) would also fit on this list. If I remember correctly, both topics were planned for 3.0 when I was in one of the calls.

**Comment by kestewart** [comment]
Yes,  those should be added to the list. 

As soon as the tech team, gets the core profile stabilized, we'll be doing another pass on the licensing profile, and these pending topics will be discussed.    If a subteam wants to meet to discuss the licensing issues, in parallel,  that would be welcome.

**Comment by goneall** [comment]
@swinslow - Can you take a pass at this list and see if we got everything?

**Comment by swinslow** [comment]
@goneall Yes, thanks -- please see my edits in the tracking comment above, and in a number of the linked issues that I've commented on or closed.

I believe everything is now addressed that is going to be addressed for 3.0.

There are 4 issues noted above that are likely still to be discussed for 3.1 (though I'm not sure what the timeframe for 3.1 is expected to be, but leaving those open for now).

I'm going to go ahead and close this tracker since it's keyed to 3.0, but feel free to re-open or let me know if you have any questions.
